src/tests/load-function-module.test.ts

```diff
@@ -49,14 +56,12 @@ Deno.test("LoadFunctionModule function", async (t) => {
   });
 
   await t.step("should throw if does not exist", async () => {
-    await assertRejects(
-      async () => {
-        return await LoadFunctionModule(
-          functionsDir,
-          generatePayload("nonexistent"),
-        );
-      },
-      Error,
+    const jsModule = await LoadFunctionModule(
+      [`${functionsDir}/nonexistnent.ts`],
```

Comment:
nit: It literally doesn't matter at all, but this nonexistent filename has a typo 😂

Reply:
🤣 feels slightly appropriate then